### DISCUSSION TITLE
Snapcat on arm64

### DISCUSSION THREAD

**Discussion by mhzawadi** [snapcast/snapcast#877] [Ideas]
Hello,

Thanks for this very amazing bit of software, I cant believe how well this works. Even over a mobile network.

What I would like to know, is if you have time to look at building for arm64?

I have started to move my pi installs over and was about to move my office snapclient when I spotted that I can install it on arm64

**Comment by gtsteff** [comment]
compile and build instructions here work : https://github.com/badaix/snapcast/blob/master/doc/build.md#linux-native
my pi 4 compiled build: https://drive.google.com/file/d/1yH29TJFo1p-opL3IUpT_DwpoY5xVxPLm/view?usp=sharing

Note: you may need to update your dependencies on arm64 to support this.
